test/NuGet.Core.Tests/NuGet.ProjectModel.Test/DependencyGraphSpecTests.cs

```diff
@@ -525,6 +525,78 @@ public void DependencyGraphSpec_AddProjectWithCentralVersionDependencies_Depende
             Assert.True(tfms[0].Dependencies.Where(d => d.Name == "foo").First().VersionCentrallyManaged);
         }
 
+        [Theory]
+        [InlineData(null)]
+        [InlineData("")]
```

Comment:
Depending on your answer to my whitespace question, you may want to add a whitespace case or two, here.